docs/DeveloperGuide.md

```diff
@@ -74,10 +74,10 @@ Named Arguments:
 ```
 --help, -h           Show help message.
 --view [PATH], -v [PATH]
-                     Starts a server to display the dashboard in the
+                     Starts a server to display the report in the
                      provided directory. If used as a flag (with no
                      argument), generates a report and automatically
-                     displays the dashboard.
+                     displays the report.
```

Comment:
Do change the help message in Argsparser as well. This message is generated from the Argsparser.

Reply:
Got it. I also updated the logger message in `DashboardServer`.